tests/test_models.py

```diff
@@ -12,17 +14,25 @@
 def as_ops():
     client = ASnakeClient(baseurl='mock://example.com', username='test',
                           password='test')
-    as_ops = models.AsOperations(client)
+    as_ops = models.AsOperations(client, logger)
     return as_ops
 
 
-def test_get_record(as_ops):
+@pytest.fixture
+def logger():
```

Comment:
What I would recommend at this point is to create a new file in your tests directory called `conftest.py`. There is some pytest magic that will load this file before tests are run. This is a good place to put fixtures that may be used in different test modules. This is where you should configure your logger fixture. There are a few changes I'd make to this as well. We probably always want the same logging configuration for testing, and we know we always want the logger configured, so we can tell pytest to do this once automatically, without us having to actually use the fixture:
```python
@pytest.fixture(autouse=True, scope='session')
def logging():
    logging.basicConfig(format="%(message)s",
                        handlers=[logging.StreamHandler()], level=logging.INFO)
```
This way, you don't have to include the logger as a fixture in all of your tests. If for some reason, you need access to the logger in your tests, you can just do like you've done in the code and add a call to `get_logger` at the test module level.

Reply:
I had to add a bit more code to get this to work but let me know if there's a more efficient way to do it